draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -128,7 +128,7 @@ contributor:
     email: dacian.demeter@telus.com 
 
 normative:
-  OIF-CMIS:
+  CMIS:
```

Comment:
This Implementation Agreement (IA) defines the Common Management Interface
Note that CMIS targets any pluggable modules, not only Coherent ones

Abstract: Specification (CMIS), which may be used by pluggable or on-board modules, such as
QSFP Double Density (QSFP-DD), OSFP, COBO, QSFP, as well as by existing or future
module developments with host to module management communication based on a
two-wire interface. This IA is targeted for systems manufacturers, system integrators,
and suppliers of CMIS compliant modules.

Reply:
Gert, that is correct.
I will add your suggested text to the document.